src/oscar/static/oscar/js/oscar/RelatedObjectLookups.js

```diff
@@ -0,0 +1,128 @@
+// Handles related-objects functionality: Add Another links.
+
+var oscar = (function(o, $) {
```

Comment:
This will clobber the global variable of the same name that is defined in dashboard.js. Probably better to inject into that variable.